requirements/requirements.txt

```diff
@@ -44,4 +44,5 @@ urllib3==1.24.2
 xlrd==1.0.0
 jsonpickle==0.9.6
 filechunkio==1.8
-pip==19.1.1
+pip==19.2.2
```

Comment:
was this intended? If so, why?

Reply:
Yes.
The real question is: Why not?

It's good to keep pip up-to-date since installing compromised packages (or having the install hijacked mid-flight) is a potential attack vector